pkg/controller/operandrequest/reconcile_operator.go

```diff
@@ -49,6 +49,14 @@ func (r *ReconcileOperandRequest) reconcileOperator(requestInstance *operatorv1a
 			// Check the requested Operand if exist in specific OperandRegistry
 			opt := registryInstance.GetOperator(operand.Name)
 			if opt != nil {
+				if opt.Scope == operatorv1alpha1.ScopePrivate && requestInstance.Namespace != registryInstance.Namespace {
+					klog.V(2).Infof("Operator %s is a private. It can't be requested from namespace %s", operand.Name, requestInstance.Namespace)
```

Comment:
warning message?

Reply:
@gyliu513 Thanks! I update it to a warning in the following commit.